From e011674130068daaa4272a2dfb129f54331438fe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Tue, 12 Nov 2019 14:52:37 +0100
Subject: [PATCH 07/16] user32/tests: Wait a bit more for ScrollWindowEx
 events.

It sometimes fails with an empty sequence on w2003std.

  https://testbot.winehq.org/JobDetails.pl?Key=59878#k102
---
 dlls/user32/tests/msg.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dlls/user32/tests/msg.c b/dlls/user32/tests/msg.c
index 11111111111..11111111111 100644
--- a/dlls/user32/tests/msg.c
+++ b/dlls/user32/tests/msg.c
@@ -12192,6 +12192,7 @@ static void test_scrollwindowex(void)
     ok_sequence(WmEmptySeq, "ScrollWindowEx", FALSE);
     if (winetest_debug > 1) trace("end scroll\n");
     flush_sequence();
+    MsgWaitForMultipleObjects( 0, NULL, FALSE, 1000, QS_ALLINPUT );
     flush_events();
     ok_sequence(ScrollWindowPaint1, "ScrollWindowEx", FALSE);
     flush_events();
@@ -12203,6 +12204,7 @@ static void test_scrollwindowex(void)
     ok_sequence(WmEmptySeq, "ScrollWindowEx", FALSE);
     if (winetest_debug > 1) trace("end scroll\n");
     flush_sequence();
+    MsgWaitForMultipleObjects( 0, NULL, FALSE, 1000, QS_ALLINPUT );
     flush_events();
     ok_sequence(ScrollWindowPaint2, "ScrollWindowEx", FALSE);
     flush_events();
@@ -12217,6 +12219,7 @@ static void test_scrollwindowex(void)
     ok_sequence(WmEmptySeq, "ScrollWindowEx", TRUE);
     if (winetest_debug > 1) trace("end scroll\n");
     flush_sequence();
+    MsgWaitForMultipleObjects( 0, NULL, FALSE, 1000, QS_ALLINPUT );
     flush_events();
     ok_sequence(ScrollWindowPaint1, "ScrollWindowEx", FALSE);
     flush_events();
@@ -12227,6 +12230,7 @@ static void test_scrollwindowex(void)
     ScrollWindow( hwnd, 5, 5, NULL, NULL);
     if (winetest_debug > 1) trace("end scroll\n");
     flush_sequence();
+    MsgWaitForMultipleObjects( 0, NULL, FALSE, 1000, QS_ALLINPUT );
     flush_events();
     ok_sequence(ScrollWindowPaint1, "ScrollWindow", FALSE);
 
-- 
2.40.0

